backend/test/mastodon/notifications.spec.ts

```diff
@@ -117,7 +117,7 @@ describe('Mastodon APIs', () => {
 
```

Comment:
Sorry to be annoying but I am not keen on these vague `any` types, even in tests.
How about:

```ts
			globalThis.fetch = async (input: RequestInfo, data?: RequestInit) => {
				assert(data)
				const headers = new Headers(data.headers)
				if (input === 'https://push.com') {
					assert(headers.get('Authorization')?.includes('WebPush'))

					const cryptoKey = headers.get('Crypto-Key')
					assert(cryptoKey)
					const cryptoKeyHeader = parseCryptoKey(cryptoKey)
					assert(cryptoKeyHeader.dh)
					assert(cryptoKeyHeader.p256ecdsa)

					// Ensure the data has a valid signature using the client public key
					assert(data.body)
					const body = data.body as ArrayBuffer | ArrayBufferView
					const sign = await crypto.subtle.sign({ name: 'ECDSA', hash: 'SHA-256' }, clientKeys.privateKey, body)
					assert(await crypto.subtle.verify({ name: 'ECDSA', hash: 'SHA-256' }, clientKeys.publicKey, sign, body))

```

Reply:
ah no I totally agree, but if it's ok I would leave this to a separate PR, in this one I am just trying to enable those handful of eslint rules you can see in the eslintrc change.

For the `any`s we'll need to enable this one:
https://github.com/cloudflare/wildebeest/blob/4a6bf1739b2df758784b44625beebc16b8234600/.eslintrc.cjs#L36
which unfortunately (as I saw then I gave it a quick shot some time ago) I think it's going to be the hardest rule to enable 😓